akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -271,6 +271,149 @@ Therefore after creating the Actor system with the `main` Actor’s
 `Behavior` the only thing we do is await its termination.
 
 
```

Comment:
Should we move it to a separate page, to avoid create huge doc pages again.
It could have a short mention and link from here.

Reply:
ok, it might also be good to not give the impression that one is favored over the other (although we have an opinion). Especially for Java developers the object oriented variant will feel more familiar (I have heard).

Could we put both under the heading "A More Complex Example" and starting with that there is an option of these two styles, with links to the sub-sections for immediate fast forward to the preferred sample (probably also good to have them nearby for side-by-side comparison).

* A More Complex Example
    * Functional style
    * Object-oriented style

I'm thinking that Object-oriented is more well known what it is about than "Class based" and most people already know in what camp they are in.

Reply:
In the light of the recent discussions around that theme I agree. Proposed structure sounds good, I'll do that.